### PR TITLE
feature: Add clang-tidy to tools that support Objective C CY-4795

### PR DESCRIPTION
**Pull request by franciscodua**

### CODE REVIEW

File: docs/getting-started/supported-languages-and-tools.md [hunk 1]
```diff
@@ -200,7 +200,8 @@ The table below lists all languages and frameworks that Codacy supports and the
     </tr>
     <tr>
       <td>Objective-C</td>
-      <td><a href="http://fauxpasapp.com/">Faux Pas</a><a href="#client-side"><sup>1</sup></a></td>
+      <td><a href="http://fauxpasapp.com/">Faux Pas</a><a href="#client-side"><sup>1</sup>,
+          <a href="https://clang.llvm.org/extra/clang-tidy/">Clang-Tidy</a><a href="#client-side"><sup>1</sup></a></td>
```
Comment (prcr) [inline docs/getting-started/supported-languages-and-tools.md:204]: Just a very minor nitpick, we're currently listing the tools in alphabetical order - in a way, this tries to convey that no tool for each language is preferred against the others. So I would include Clang-Tidy first:

```suggestion
      <td><a href="https://clang.llvm.org/extra/clang-tidy/">Clang-Tidy</a><a href="#client-side"><sup>1</sup>,
          <a href="http://fauxpasapp.com/">Faux Pas</a><a href="#client-side"><sup>1</sup>
          </a></td>
```

**Review by prcr — COMMENTED** [review]
Thanks for the pull request @franciscodua! :+1:

File: docs/getting-started/supported-languages-and-tools.md (hunk 1 above)
Comment (franciscodua) [reply docs/getting-started/supported-languages-and-tools.md]: I amended the commit with this change ✅

**Review by lolgab — APPROVED** [review]

**Review by prcr — APPROVED** [review]
:shipit:

**Pull request merged**